internal/run/run.go

```diff
@@ -304,6 +309,18 @@ func (w *Workflow) runTarget(ctx context.Context, target string) error {
 	}
 	w.generationAccess = generationAccess
 
+	if t.CodeSamples != nil {
+		rootStep.NewSubstep("Generating CodeSamples")
+		configPath := "."
+		if t.Output != nil {
+			configPath = *t.Output
```

Comment:
It's a bit ambiguous just looking at the workflow config if the `codeSamples output` path is relative to the target output path or not. I suspect it should be

Reply:
Hmm good point. It's not target output path right now, but it should be I think. I'll change that.